CONTRIBUTING.md

```diff
@@ -1,19 +1,9 @@
-# Contributing guidelines
+# Contributing
 
-We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
+Welcome to Kubernetes! To learn more about contributing to the [Kubernetes code repo](README.md), check out the [Contributor's Guide](https://git.k8s.io/community/contributors/guide/).
```

Comment:
The link to the README as well as the mention of Kubernetes code repo does seems incorrect.
```suggestion
Welcome to git-sync a project within the Kubernetes ecosystem! To learn more about contributing check out the [general Contributor's Guide](https://git.k8s.io/community/contributors/guide/).
```